tests/general/mmap.c

```diff
@@ -0,0 +1,87 @@
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <limits.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#ifdef __GLIBC__
+#include <sys/user.h>
+#endif
+#include <unistd.h>
+
+#define perror_and_exit(message) \
+  do {                           \
+    perror(message);             \
+    return EXIT_FAILURE;         \
+  } while (0)
+
+#define OFFSET "10726"
+#define LENGTH "143"
```

Comment:
Why parse these strings?  Why not just `static const size_t LENGTH` and `static const off_t OFFSET`?  Maybe they were originally argv string?  Seems like this test code would be a lot more precise and concise without the parsing and validation.

Reply:
Done. Yeah, these are essentially parameters. I've now converted them to integers to avoid the atoll/strtoll.